build/arm/linker/linker_nrf52840_common.ld

```diff
@@ -1,22 +1,17 @@
+/* Default stack sizes. These are used by the startup in order to allocate stacks for the
```

Comment:
This file is only referenced by bootloader, would it be better to rename it to `linker_nrf52840_bootloader_common.ld`?